src/lib.rs

```diff
@@ -210,6 +213,96 @@ macro_rules! wrap_pymodule {
     }};
 }
 
+/// A convinient macro to execute a Python code snippet, with some local variables set.
+///
+/// # Example
+/// ```
+/// use pyo3::{prelude::*, py_run, types::PyList};
+/// let gil = Python::acquire_gil();
+/// let py = gil.python();
+/// let obj = pyo3::types::PyDict::new(py);
+/// let list = PyList::new(py, &[1, 2, 3]);
```

Comment:
nit: it's kind of inconsistent to import `PyList` but use `PyDict` fully-qualified...